contracts/bonding/BondingCheckpoints.sol

```diff
@@ -0,0 +1,194 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
```

Comment:
Yeah ideally we should, but upgrading this was a real rabbit hole. Since our project is already using 0.8.9 for now, we'd need to upgrade all contracts to accept the new version or at least `^0.8.9` (not fixed) instead to be able to create a new contract that references them. This would be so much changes that I gave up and just kept them as 0.8.9 for now.